R/tag_query.R

```diff
@@ -762,23 +769,20 @@ tagQuery_ <- function(
         #' [`tagAppendChildren()`]
         append = function(...) {
           tagQueryChildrenAppend(selected_, ...)
-          rebuild_()
```

Comment:
IIUC these aren't really needed since `tagQueryChildren*()` winds up calling `asTagEnv(el)`?

Reply:
Correct. I missed removing them in #235